scipy/optimize/tests/test_linprog.py

```diff
@@ -383,7 +383,7 @@ def test_disp(self):
         A, b, c = lpgen_2d(20, 20)
         res = linprog(c, A_ub=A, b_ub=b, method=self.method,
                       options={"disp": True})
-        _assert_success(res, desired_fun=-64.049494229)
+        _assert_success(res, desired_fun=-63.47967608020187)  # method='highs' solution
```

Comment:
I don't remember whether there was a source for these values. In any case, when the tests were written, 'highs'` didn't exist, so I think it's reasonable to use it as a source of ground truth for all the other methods. And if all the other methods agree with it, it's probably right, too.